src/Pmi/Console/Command/MetricsCommand.php

```diff
@@ -66,12 +67,23 @@ protected function configure()
                 InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
                 'Filter to specified statuses.'
             )
+            ->addOption(
+                'history',
+                null,
+                InputOption::VALUE_NONE,
+                'Return cached history of data; required for certain stratifications.'
+            )
             ->addOption(
                 'pretty',
                 null,
-                InputOption::VALUE_OPTIONAL,
-                'Pretty formatting for JSON ouput',
-                false
+                InputOption::VALUE_NONE,
+                'Pretty formatting for JSON ouput'
+            )
+            ->addOption(
+                'debug',
+                null,
+                InputOption::VALUE_NONE,
+                'Show request debug',
```

Comment:
Remove trailing comma. Isn't this is a syntax error in PHP < 7.3? Why doesn't this throw an error for you when running using the App Engine runtime?

```suggestion
                'Show request debug'
```

Reply:
Commands are run locally, so I didn't encounter that.